assets/javascripts/discourse/components/query-result.hbs

```diff
@@ -1,76 +1,77 @@
 <article>
   <header class="result-header">
     <div class="result-info">
-      {{d-button
-        action=(action "downloadResultJson")
-        icon="download"
-        label="explorer.download_json"
-        group=group
-      }}
+      <DButton
+        @action={{this.downloadResultJson}}
+        @icon="download"
+        @label="explorer.download_json"
+        @group={{this.group}}
```

Comment:
(I realise you didn't introduce it... but...)

I don't recognize the `@group` argument to `DButton`, and I don't see any reference to it in core. Any idea what it does? If not, maybe it should be removed? 🤔

Reply:
It is not being used, I will drop it.